_posts/Redis/1300-01-01-Flushall-Redis.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2018-05-03
+title: Redis's Flushall Is Not Spinnaker's Friend
```

Comment:
I'd change the title to something a little more descriptive. Something like "When to use FLUSHALL on Spinnaker's Redis Cache".